### PR TITLE
Update palo-alto-firewall-base-document-Version10

### PR DESCRIPTION
**Pull request by mvirga01**
This proposed change, updates the parent Palo Alto Firewall ingestion doc for support to installed collector (version 10)

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by MariePMcGarry — APPROVED** [review]

File: docs/cse/ingestion/palo-alto-firewall.md [hunk 1]
```diff
@@ -50,11 +50,62 @@ In this step, you configure a Syslog Source to collect Palo Alto Firewall log me
...
+   * Name: `Sumo_CloudSyslog_EndPoint01`
+   * Syslog Server: Internal IP of Collector
+   * Transport: UDP
+   * Port: Port from Collector Setup
+   * Format: BSD
+   * Facility: `LOG_USER`
+6. In the **Syslog Server Profile** window, select the **Custom Log Format** tab, and use the following custom formats for the following log types:
+
+[Config](https://docs.paloaltonetworks.com/pan-os/10-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/config-log-fields)
+,$receive_time,$serial,$type,$subtype,,$time_generated,$host,$vsys,$cmd,$admin,$client,$result,$path,$before-change-detail,$after-change-detail,$seqno,$actionflags,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,$vsys_name,$device_name<Token from Step 1>
```
Comment (kimsauce) [inline docs/cse/ingestion/palo-alto-firewall.md:73]: Missing code block formatting. Needs three backticks before and after.

```suggestion
,$receive_time,$serial,$type,$subtype,,$time_generated,$host,$vsys,$cmd,$admin,$client,$result,$path,$before-change-detail,$after-change-detail,$seqno,$actionflags,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,$vsys_name,$device_name<Token from Step 1>
```

File: docs/cse/ingestion/palo-alto-firewall.md [hunk 2]
```diff
@@ -50,11 +50,62 @@ In this step, you configure a Syslog Source to collect Palo Alto Firewall log me
...
+
+
+[System](https://docs.paloaltonetworks.com/pan-os/10-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/system-log-fields)
+,$receive_time,$serial,$type,$subtype,,$time_generated,$vsys,$eventid,$object,,,$module,$severity,$opaque,$seqno,$actionflags,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,$vsys_name,$device_name,$high_res_timestamp
```
Comment (kimsauce) [inline docs/cse/ingestion/palo-alto-firewall.md:77]: Missing code block formatting. Needs three backticks before and after.

```suggestion
,$receive_time,$serial,$type,$subtype,,$time_generated,$vsys,$eventid,$object,,,$module,$severity,$opaque,$seqno,$actionflags,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,$vsys_name,$device_name,$high_res_timestamp
```

File: docs/cse/ingestion/palo-alto-firewall.md [hunk 3]
```diff
@@ -50,11 +50,62 @@ In this step, you configure a Syslog Source to collect Palo Alto Firewall log me
...
+,$receive_time,$serial,$type,$subtype,,$time_generated,$src,$dst,$natsrc,$natdst,$rule,$srcuser,$dstuser,$app,$vsys,$from,$to,$inbound_if,$outbound_if,$logset,,$sessionid,$repeatcnt,$sport,$dport,$natsport,$natdport,$flags,$proto,$action,$bytes,$bytes_sent,$bytes_received,$packets,$start,\$elapsed,$category,,$seqno,$actionflags,$srcloc,$dstloc,,$pkts_sent,$pkts_received,$session_end_reason,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,$vsys_name,$device_name,$action_source,$src_uuid,$dst_uuid,$tunnelid/$imsi,$monitortag/$imei,$parent_session_id,$parent_start_time,$tunnel,$assoc_id,$chunks,$chunks_sent,$chunks_received,$rule_uuid,$http2_connection,$link_change_count,$policy_id,\$link_switches,$sdwan_cluster,$sdwan_device_type,$sdwan_cluster_type,$sdwan_site,$dynusergroup_name,$xff_ip,$src_category,$src_profile,$src_model,$src_vendor,$src_osfamily,$src_osversion,$src_host,$src_mac,$dst_category,$dst_profile,$dst_model,$dst_vendor,$dst_osfamily,$dst_osversion,$dst_host,$dst_mac,$container_id,$pod_namespace,$pod_name,$src_edl,$dst_edl,$hostid,$serialnumber,$src_dag,$dst_dag,$session_owner,\$high_res_timestamp,$nsdsai_sst,$nsdsai_sd <Token from Step 1>
```
Comment (kimsauce) [inline docs/cse/ingestion/palo-alto-firewall.md:82]: Missing code block formatting. Needs three backticks before and after.

```suggestion
,$receive_time,$serial,$type,$subtype,,$time_generated,$src,$dst,$natsrc,$natdst,$rule,$srcuser,$dstuser,$app,$vsys,$from,$to,$inbound_if,$outbound_if,$logset,,$sessionid,$repeatcnt,$sport,$dport,$natsport,$natdport,$flags,$proto,$action,$bytes,$bytes_sent,$bytes_received,$packets,$start,\$elapsed,$category,,$seqno,$actionflags,$srcloc,$dstloc,,$pkts_sent,$pkts_received,$session_end_reason,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,$vsys_name,$device_name,$action_source,$src_uuid,$dst_uuid,$tunnelid/$imsi,$monitortag/$imei,$parent_session_id,$parent_start_time,$tunnel,$assoc_id,$chunks,$chunks_sent,$chunks_received,$rule_uuid,$http2_connection,$link_change_count,$policy_id,\$link_switches,$sdwan_cluster,$sdwan_device_type,$sdwan_cluster_type,$sdwan_site,$dynusergroup_name,$xff_ip,$src_category,$src_profile,$src_model,$src_vendor,$src_osfamily,$src_osversion,$src_host,$src_mac,$dst_category,$dst_profile,$dst_model,$dst_vendor,$dst_osfamily,$dst_osversion,$dst_host,$dst_mac,$container_id,$pod_namespace,$pod_name,$src_edl,$dst_edl,$hostid,$serialnumber,$src_dag,$dst_dag,$session_owner,\$high_res_timestamp,$nsdsai_sst,$nsdsai_sd <Token from Step 1>
```

File: docs/cse/ingestion/palo-alto-firewall.md [hunk 4]
```diff
@@ -50,11 +50,62 @@ In this step, you configure a Syslog Source to collect Palo Alto Firewall log me
...
+,$receive_time,$serial,$type,$subtype,,$time_generated,$src,$dst,$natsrc,$natdst,$rule,$srcuser,$dstuser,$app,$vsys,$from,$to,$inbound_if,$outbound_if,$logset,,$sessionid,$repeatcnt,$sport,$dport,$natsport,$natdport,$flags,$proto,$action,$misc,$threatid,$category,$severity,$direction,\$seqno,$actionflags,$srcloc,$dstloc,,$contenttype,$pcap_id,$filedigest,$cloud,$url_idx,$user_agent,$filetype,$xff,$referer,$sender,$subject,$recipient,$reportid,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,\$vsys_name,$device_name,,$src_uuid,$dst_uuid,$http_method,$tunnel_id/$imsi,$monitortag/$imei,$parent_session_id,$parent_start_time,$tunnel,$thr_category,$contentver,,$assoc_id,$ppid,$http_headers,$url_category_list,$rule_uuid,$http2_connection,$dynusergroup_name,$xff_ip,$src_category,$src_profile,$src_model,$src_vendor,$src_osfamily,$src_osversion,$src_host,$src_mac,$dst_category,$dst_profile,$dst_model,\$dst_vendor,$dst_osfamily,$dst_osversion,$dst_host,$dst_mac,$container_id,$pod_namespace,$pod_name,$src_edl,$dst_edl,$hostid,$serialnumber,$domain_edl,$src_dag,$dst_dag,$partial_hash,$high_res_timestamp,$reason,\$justification,$nssai_sst<Token from Step 1>
```
Comment (kimsauce) [inline docs/cse/ingestion/palo-alto-firewall.md:86]: Missing code block formatting. Needs three backticks before and after.

File: docs/cse/ingestion/palo-alto-firewall.md [hunk 5]
```diff
@@ -50,11 +50,62 @@ In this step, you configure a Syslog Source to collect Palo Alto Firewall log me
...
+
+
+[Hip Match](https://docs.paloaltonetworks.com/pan-os/10-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/hip-match-log-fields)
+,$_receive_time,$serial,$type,$subtype,,$time_generated,$srcuser,$vsys,$machinename,$os,$src,$matchname,$repeatcnt,$matchtype,,,$seqno,$actionflags,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,$vsys_name,$device_name,$vsys_id,$srcipv6,$hostid,$mac,$high_res_timestamp<Token from Step 1>
```
Comment (kimsauce) [inline docs/cse/ingestion/palo-alto-firewall.md:90]: Missing code block formatting. Needs three backticks before and after.

File: docs/cse/ingestion/palo-alto-firewall.md [hunk 6]
```diff
@@ -50,11 +50,62 @@ In this step, you configure a Syslog Source to collect Palo Alto Firewall log me
...
+
+
+[UserID](https://docs.paloaltonetworks.com/pan-os/10-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/user-id-log-fields)
+,$receive_time,$serial,$type,$subtype,,$time_generated,$vsys,$ip,$user,$datasourcename,$eventid,$repeatcnt,$timeout,$beginport,$endport,$datasource,$datasourcetype,$seqno,$actionflags,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,$vsys_name,$device_name,$vsys_id,$factortype,$factorcompletiontime,$factorno,,,$ugflags,$userbysource,$high_res_timestamp<Token from Step 1>
```
Comment (kimsauce) [inline docs/cse/ingestion/palo-alto-firewall.md:94]: Missing code block formatting. Needs three backticks before and after.

File: docs/cse/ingestion/palo-alto-firewall.md [hunk 7]
```diff
@@ -50,11 +50,62 @@ In this step, you configure a Syslog Source to collect Palo Alto Firewall log me
...
+
+[GlobalProtect](https://docs.paloaltonetworks.com/pan-os/10-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/globalprotect-log-fields)
+,$receive_time,$serial,$type,$subtype,,$time_generated,$vsys,$ip,$user,$datasourcename,$eventid,$repeatcnt,$timeout,$beginport,$endport,$datasource,$datasourcetype,$seqno,$actionflags,$dg_hier_level_1,$dg_hier_level_2,$dg_hier_level_3,$dg_hier_level_4,$vsys_name,$device_name,$vsys_id,$factortype,$factorcompletiontime,$factorno,,,$ugflags,$userbysource<Token from Step 1>
```
Comment (kimsauce) [inline docs/cse/ingestion/palo-alto-firewall.md:97]: Missing code block formatting. Needs three backticks before and after.

File: docs/cse/ingestion/palo-alto-firewall.md [hunk 8]
```diff
@@ -50,11 +50,62 @@ In this step, you configure a Syslog Source to collect Palo Alto Firewall log me
...
+
+
+7. Click OK.
+8. Commit the changes.
+
+
+## Step 3: Configure Palo Alto Firewall
+
+In this step you configure Palo Alto Firewall to send log messages to the Sumo Logic platform. Follow Palo Alto documentation to [Configure Log Forwarding](https://docs.paloaltonetworks.com/pan-os/10-1/pan-os-admin/monitoring/configure-log-forwarding).
```
Comment (kimsauce) [inline docs/cse/ingestion/palo-alto-firewall.md:106]:
```suggestion
In this step, you configure Palo Alto Firewall to send log messages to the Sumo Logic platform. Follow Palo Alto documentation to [Configure Log Forwarding](https://docs.paloaltonetworks.com/pan-os/10-1/pan-os-admin/monitoring/configure-log-forwarding). 
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**